framework/src/media/audio/audio_manager.c

```diff
@@ -1245,14 +1245,24 @@ uint32_t get_output_bytes_to_frame(unsigned int bytes)
 	return pcm_bytes_to_frames(g_audio_out_cards[g_actual_audio_out_card_id].pcm, bytes);
 }
 
-audio_manager_result_t get_max_audio_volume(uint8_t *volume)
+audio_manager_result_t get_max_audio_volume(audio_manager_stream_policy_t type, uint8_t *volume)
```

Comment:
we don't have to add policy as a parameter, card->policy already there.

Reply:
Modified.